qiskit/optimization/converters/quadratic_program_to_qubo.py

```diff
@@ -104,7 +109,7 @@ def get_compatibility_msg(problem: QuadraticProgram) -> str:
 
         # initialize message
         msg = ''
-
+        print(problem)
```

Comment:
I guess this is a left over from debugging? We should not have print() calls in the code. A logger with the logging function though can remain if it is felt this is useful to have